boxwise_flask/models.py

```diff
@@ -1,7 +1,8 @@
 """Model definitions for database"""
-from peewee import CharField
+from peewee import CharField, Model, ForeignKeyField, CompositeKey
```

Comment:
The Model and ForeignKeyField imports are not required (flake8 detects such things)